.github/workflows/build.yml

```diff
@@ -40,7 +40,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        java: [8,11]
+        java: [8,17]
```

Comment:
Have you considered dropping `8` instead of `11`?

Java 8 support already ended on 31 Mar 2022.

Reply:
We want Fabric8 to be 8 compatible.

We will probably drop its support once everyone migrates to Spring 6 :wink:, or until we can no longer sustain it, or if there's some 11 specific feature that we need. For the last part, what we've followed is the creation of JDK11 specific optional modules, which seem to work well.

Reply:
Fair enough 👍